scripts/build_openssl.sh

```diff
@@ -27,6 +27,7 @@ openssl_config_opts=()
 openssl_config_opts+=(
     "no-shared"
     "--prefix=$OPENSSL_BUILD_DIR"
+    "--openssldir=$OPENSSL_BUILD_DIR"
```

Comment:
The OpenSSL build page recommends that we set both of these parameters for the version of OpenSSL that we are using (not doing so can lead to build inconsistencies).